### PR TITLE
feat: More features for our MVP Phase 1

### LINKED ISSUE #22: Write Step-By-Step Deployment Guide
We want to write a document (DEPLOYMENT.md) where we describe the steps required to deploy the Airtable Gateway on an Airtable. 
- Includes cloning and env variable setup of repo
- Includes what fields are required on Airtable
- "Deploy to Heroku" Button?

### LINKED ISSUE #23: Write DELETE for Client Group Airtable
We need to be able to 'delete' a needs request from the client group airtable given a request ID

### LINKED ISSUE #24: Write UPDATE for Client Group Airtable
We need to be able to 'update' a needs request from the client group airtable given a request ID. We will need to search the request IDs of the table, and then update the row if it exists, else create a new row.

### LINKED ISSUE #25: Add Env Variables for Column Mapping
Right now the column mappings are hard-coded, we are going to now add these in env variables. 
If the env variable is not defined, we can fall back to hard-coded mapping.

### LINKED ISSUE #26: We shouldn't be freezing config variables
This will prevent detailed testing using mocha, we should be able to change the values as needed during testing but because of Object.freeze we can't
https://github.com/MutualAidNYC/airtable_gateway/blob/19326eb672edbc33bd742edd3194862182a8c5e4/src/config/index.js#L35

### LINKED ISSUE #29: airtable_change_detector needs to be updated
airtable_change_detector was updated to version 1.1 with better error handling.

### LINKED ISSUE #30: Begrudgingly allow for free heroku dyno
@LieseOnLife would strongly suggest we advise against using this option in production.

Reasons we are implementing:
1. So people can test the app without being charged
2. We may have future uses down the road where time is not of the essence
3. A MA org may not have any funds, any _slow_ updates are better than _no_ updates

### PR DESCRIPTION
**Pull request by brainomite**
After this PR, all developers need to do an `npm install` as a dependency was updated.

Any and all new code have Mocha tests written for it!

- [x] Remove freezing of config variables this will resolve #26
- [x] Add environment variables instead of hard-coded mapping which will resolve #25 
- [x] Write UPDATE for client table which will resolve #24 
- [x] Write DELETE for Client Group Airtable which will resolve #23
- [x] Write DEPLOYMENT.md optionally using Heroku Buttons which will resolve #22 
- [x] Update airtable_change_detector dependency which will resolve #29 
- [x] Update the system to permit free Heroku tier, begrudgingly, which will resolve #30

### CODE REVIEW

File: DEPLOYMENT.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Airtable Gateway deployment guide
+
+
+We at MANYC are presently only supporting [paid Heroku dynos](https://www.heroku.com/pricing).
+This project is an 'active' server that actively polls your airtable every 10 seconds. Unfortunately  the free Heroku tier won't let you poll for any period of time before the free Heroku dyno goes to [sleep](https://devcenter.heroku.com/articles/free-dyno-hours#dyno-sleeping)
+
+We hope in the future to have FULL deployment instructions on alternative platforms.
```
Comment (btmoore4) [inline DEPLOYMENT.md:7]: Like this section, we can def have separate sections for each type of deployment

File: DEPLOYMENT.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+   2. MANYC Update Request URL
+   3. MANY Delete Request URL
+2. Add to your list of statuses a status 'Recalled' this will be used when MANYC triggers a cancel of a request.
+3. Click this button, and continue reading.
+   [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
+4. Give your app a unique Heroku name, the name itself doesn't have an effect.
+5. Choose the 'App Owner'
+6. For region, 'United States' is sufficient.
+7. Next up we have a quantity of 'Config Vars' to fill in. These are in alphabetical order. I will briefly describe them by their prefixes.
+   1. AIRTABLE_
```
Comment (btmoore4) [inline DEPLOYMENT.md:26]: Just a typo having this underscores after the variable?

File: src/jobs/pollOnce.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+'use strict';
+
```
Comment (btmoore4) [inline src/jobs/pollOnce.js:2]: Does this need to be a separate file?

File: DEPLOYMENT.md (hunk 2 above)
Comment (brainomite) [reply DEPLOYMENT.md:26]: No, intentional

File: src/jobs/pollOnce.js (hunk 3 above)
Comment (brainomite) [reply src/jobs/pollOnce.js:2]: Yes, it's being executed directly by node as a cron job

**Pull request merged**